src/plotman/archive.py

```diff
@@ -130,13 +140,41 @@ def archive(dir_cfg, all_jobs):
         (archdir, freespace) = sorted(available)[index]
 
     if not archdir:
-        return(False, 'No archive directories found with enough free space')
-    
+        return 'No archive directories found with enough free space'
+
+    arch_jobs = get_running_archive_jobs(dir_cfg['archive'])
+    if arch_jobs:
+        # Build status list
+        status_list = 'Cannot retrieve status.'
+        for j in jobs:
+            status_list = ''
+            try:
+                line = j[0].get_nowait() # or q.get(timeout=.1)
+            except Empty:
+                line = 'No data yet.'
+            if (line is None) or (line in j[1]):
+                line = 'No data yet.'
+            status_list += 'Moving ' + j[1] + ' to ' + j[2] + '. Progress: ' + line + '\n'
```

Comment:
Why reassign `status_list` to empty string at the beginning of each for loop when you're trying to append to it with new status every time?

Reply:
Thank you for the feedback! I have made some untested changes, will resume tomorrow and test when I have a plot available then perhaps we can merge.

Reply:
Sounds good! And thank you for tackling 😁